src/nix/doctor.cc

```diff
@@ -36,13 +38,16 @@ struct CmdDoctor : StoreCommand
         auto type = getStoreType();
 
         if (type < tOther) {
-            checkNixInPath();
-            checkProfileRoots(store);
+            success &= checkNixInPath();
+            success &= checkProfileRoots(store);
         }
-        checkStoreProtocol(store->getProtocol());
+        success &= checkStoreProtocol(store->getProtocol());
+
+        if (!success)
+            throw Exit(2);
```

Comment:
Is there something other than the description I can use to document this exit code?

Reply:
No, the `nix` command does not have a man page at the moment.

Reply:
It would be nice to have something to provide a longer description that only shows up with `nix <command> --help` similar to the examples.